tests/unix/subclass_native.py

```diff
@@ -1,52 +0,0 @@
-try:
-    NativeBaseClass
```

Comment:
I added this test. I think we want to keep it.

Reply:
I deleted all the `tests/unix` by accident. Restored!